floodviz/views.py

```diff
@@ -18,8 +17,32 @@ def home():
 
     j = hydrograph_utils.req_hydrodata(sites, start_date, end_date, url_top)
     all_series_data = hydrograph_utils.parse_hydrodata(j)
-    
-    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout: 
+
+    with open('floodviz/static/data/hydrograph_data.json', 'w') as fout:
         json.dump(all_series_data, fout, indent=1)
 
     return render_template('index.html')
+
+
+@app.route('/map/')
+def sitemap():
+    site_data = map_utils.site_dict(app.config['SITE_IDS'], app.config['NWIS_SITE_SERVICE_ENDPOINT'])
+    site_data = map_utils.create_geojson(site_data)
+    projection = map_utils.projection_info(app.config['PROJECTION_EPSG_CODE'], app.config['SPATIAL_REFERENCE_ENDPOINT'])
+
+    # TODO: the following is a hack:
+    with open('instance/counties.json', 'r') as bg_file:
+        bg_data = json.load(bg_file)
+    # End hack
+
+
+    mapinfo = {
+        'proj4string': projection,
+        'width': 1000,
+        'height': 600,
+        'scale': .90,  # larger than .97 cuts off data
```

Comment:
Is this comment true just for this projection? is this dependent on the bounding box or width/height? Also remember that the aspect ratio and width will be specified in config.py and should not be hard coded.